### PR TITLE
new icon: hibernate (original, original-wordmark, plain, plain-wordmark)

### PR DESCRIPTION
**Pull request by kilian-paquier**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes NONE**

## Link to prove your SVG is correct and up-to-date.

https://hibernate.org/

### CODE REVIEW

**Review by Snailedlt — APPROVED** [review]
LGTM!

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Nvm, this PR has merge conflicts which need to be solved before merging

**Review by Snailedlt — COMMENTED** [review]
As far as I can tell this is a brand, and not a technology. So it doesn't meet the requirements for devicons
Screenshot_20220717_164630

**Comment by kilian-paquier** [comment]
https://www.jmdoudoux.fr/java/dej/chap-hibernate.htm
https://www.tutorialspoint.com/hibernate/orm_overview.htm
https://en.wikipedia.org/wiki/Hibernate_(framework)

The only difference between nhibernate and hibernate is the target language.

**Comment by Snailedlt** [comment]
@kilian-paquier 
The wikipedia link you sent calls it Hibernate ORM.

Okey, so hibernate is the company/organization name, and hibernate ORM is the product name. But then the product is also called hibernate (quite confusing if you ask me).

So then, shouldn't the name be `hibernateorm`? We can have `hibernate` as an alt name too

**Comment by kilian-paquier** [comment]
Hibernate is a framework, it seems to be owned by Red Hat, but it's open source. Hibernate provides functionalities/libraries/tools, such as the hibernate ORM, validator, etc.

All the functionalities come with the same logo.

Here's the GitHub link of hibernate : https://github.com/hibernate

**Comment by Snailedlt** [comment]
Yeah, looking at their twitter account it says something like "...hibernate is a suite of open-source projects..."

@Panquesito7 Looks like this is a suite of products (not a single one), do you think it qualifies for devicons?

**Review by Snailedlt — APPROVED** [review]
Approved by me, but will wait for other maintainers to see if they think it's an icon that should be accepted or not.

@devicons/maintainer Please take a look at the comments on this PR

File: devicon.json
Comment (Snailedlt) [inline devicon.json:7313]: The object needs to be moved so it's in alphabetical order in relation to the other icons

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Review by Snailedlt — DISMISSED** [review]
Thanks for your contribution! :bow:

File: devicon.json
Comment (Panquesito7) [inline devicon.json:3062]: Isn't this Java related, too?
```suggestion
            "framework",
            "java"
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by Panquesito7 — APPROVED** [review]
Awesome! Thanks for contributing! 🚀 


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Maintainable

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**
